src/passes/hash-stringify-walker.cpp

```diff
@@ -269,7 +269,8 @@ std::vector<SuffixTree::RepeatedSubstring> StringifyProcessor::filterBranches(
   const std::vector<Expression*>& exprs) {
   return StringifyProcessor::filter(
     substrings, exprs, [](const Expression* curr) {
-      return Properties::isBranch(curr) || curr->is<Return>();
+      return Properties::isBranch(curr) || curr->is<Return>() ||
+             curr->is<TryTable>();
```

Comment:
@aheejin, separately from this PR, I think it would make sense for `Properties::isBranch()` to be `true` for `TryTable` expressions. Is that something you've thought about?